tests/test_driver.py

```diff
@@ -181,6 +181,29 @@ def test_detailed_transactions_with_parents(self, mock_get_categories):
         self.assertTrue('parentCategoryName' in results_with_parents)
         self.assertTrue('parentCategoryId' in results_with_parents)
 
+
+    @patch.object(mintapi.Mint, '_get_api_key_header')
+    @patch.object(mintapi.Mint, '_load_mint_credit_url')
+    @patch.object(mintapi.api, 'get_web_driver')
+    @patch.object(mintapi.Mint, '_get_credit_reports')
+    @patch.object(mintapi.Mint, 'get_credit_inquiries')
+    @patch.object(mintapi.Mint, 'get_credit_accounts')
+    @patch.object(mintapi.Mint, 'get_credit_utilization')
+
+    def test_exclude_inquiries(self, mock_get_api_key_header, mock_load_mint_credit_url, mock_driver, mock_get_credit_reports, mock_get_credit_inquiries, mock_get_credit_accounts, mock_get_credit_utilization):
```

Comment:
I'm not saying it's wrong but... what is the intended scope of this test? Seems like you could easily just mock `get_credit_accounts` and `get_credit_utilization` and the assert would still work. Seems like a lot of extra set up.

Reply:
@jaron-l the scope of this test is to verify that when passing `exclude-inquiries` that the resulting object does not return.  I'll take a look again and see if I can simplify the test, but because there are so many requests associated with getting credit detail, it needed several mocks.  But let me see what I can do.

Reply:
@jaron-l I looked at this again.  Looks like I can remove the mock of `get_credit_inquiries` since we're excluding that data anyways, so thanks for that assist.  All of the others are necessary, though, since we are accessing another URL and so there is more that goes into the `get_credit_report` method compared to others.

Reply:
Could you just mock the `Mint` class and assert that `get_credit_inquires` was not called? Would simplify the code a lot. Also, I noticed that your patching is backwards. Python decorators work closest to function act first, i.e. in your case, your patch for `get_credit_utilization` will show up in the first argument where you have `mock_get_api_key_header`.

Reply:
Ah nevermind, that won't work. Of you could just fix the mocks and if I can't think of a clearer way we'll leave it.

Reply:
@jaron-l fixed the order of the mocks.

Reply:
I noticed that you don't actually modify any of the mock objects so you can probably delete all the `return_value` modifications. I'll go ahead and do that, push, and merge.